frontend/src/pages/add-blog.tsx

```diff
@@ -42,7 +50,17 @@ function AddBlog() {
       });
     }
   };
-
+  const handleselector = () => {
+    var url = "";
+    if (typeof selectedImage === 'string') {
+      url = selectedImage;
```

Comment:
Removing null should help us removing this overhead condition

Reply:
Done, removed